src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -167,6 +167,18 @@ private void testSubmitAction() throws Exception {
         submitPage.fillResponseTextBox(18, 0, 0, "90");
         submitPage.fillResponseTextBox(18, 0, 1, "10");
 
+        // total sums up to expected value, verify empty entries are filled with 0
+        submitPage.fillResponseTextBox(19, 1, 0, "200");
+        assertEquals(submitPage.getResponseTextBoxValue(19, 0, 0), "0");
+
+        // delete an auto-filled 0, verify it's auto-filled again
+        submitPage.clearResponseTextBoxValue(19, 0, 0);
+        assertEquals(submitPage.getResponseTextBoxValue(19, 0, 0), "0");
+
+        // after auto-filled with 0, modify a non-zero value and remove a 0,
+        submitPage.fillResponseTextBox(19, 0, 0, "90");
+        submitPage.fillResponseTextBox(19, 1, 0, "110");
```

Comment:
I don't think this is the test that @damithc had in mind. I believe the intention was to modify a non-zero value such that the sum of values is no longer correct (ie. not 200 in this case), and then remove an existing 0 (ie. change it to an empty value). Finally, check that the empty field is not automatically filled in with a 0.